frontend/src/pages/Transactions/components/BridgeInput.tsx

```diff
@@ -168,6 +176,7 @@ const BridgeInput = ({bridgeTxFee, getCardanoTxFee, getEthTxFee, submit, loading
             {/* validate inputs */}
             <PasteApexAmountInput
                 maxSendableDfm={maxAmountDfm}
+                maxWrappedAmount={maxWrappedAmount}
```

Comment:
I dont think you need to send both to <PasteApexAmountInput/>, since that input handles only one value. You can maybe rename maxSendableDfm to maxSendable, and then pass to it, either maxAmountDfm or maxWrappedAmount, based on the selected token